Make docs more clear (#149)
DavidXanatos/sandboxie-docs: PlusContent/imdisk.md

# ImDisk

> __ImDisk__ creates a fake hard drive inside your computer's memory this is called ramdisk so after you terminate the sandbox, everything in the ramdisk will be deleted.

Reviewer: ```suggestion
__ImDisk__ creates a fake hard drive called RAM disk from your computer's memory, so after you terminate the sandbox, everything in the RAM disk will be deleted.
```

Reviewer: I don't use ImDisk, but this sentence looks like it's mandatory to delete the RAM disk content after terminating the sandbox. This should be explained better for those who are not using the feature. Maybe @offhub could help with this.
